quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -395,9 +393,10 @@ impl CongestionController for BbrCongestionController {
         self.handle_lost_packet(lost_bytes, packet_info, random_generator, timestamp);
     }
 
-    fn on_explicit_congestion(&mut self, ce_count: u64, event_time: Timestamp) {
+    fn on_explicit_congestion(&mut self, ce_count: u64, _event_time: Timestamp) {
```

Comment:
why is _event_time not used anymore? can you simply remove it?

Reply:
Initially I had an ECN notification updating the `recovery::State` (entering and exiting recovery), as that is the recommendation from RFC9002 for loss based congestion controllers. After looking at the Linux TCP BBR implementation and Chromium, neither of them consider ECN notifications for managing recovery state, so I changed it to match those implementations. Cubic still uses `event_time` from the CongestinoController trait though.

I can add a comment explaining this

Reply:
_Actually_, looking into this further, I may have been confused looking at BBRv1 code in Chromium (which doesn't consider ECN for anything). BBRv2 in Chromium doesn't seem to implement recovery yet ([link](https://source.chromium.org/chromium/chromium/src/+/main:net/third_party/quiche/src/quiche/quic/core/congestion_control/bbr2_sender.cc;l=468;bpv=1;bpt=1)). And I do see indications that TCP Linux [updates recovery state](https://github.com/google/bbr/blob/v2alpha/net/ipv4/tcp_input.c#L2901) based on ECN.

I'll bring back updating the recovery state based on ECN.